exercises/diffie-hellman/.meta/src/reference/java/DiffieHellman.java

```diff
@@ -0,0 +1,24 @@
+import java.math.BigInteger;
+import java.util.Random;
+
+public class DiffieHellman {
```

Comment:
As per our [POLICIES](https://github.com/exercism/java/blob/master/POLICIES.md#adhere-to-best-practices) we should restrict access as much as possible. Part of this would be making the class declaration package-private 🙂

Reply:
Yes, sorry. I'm going to remove the ```public``` modifier 😅